doc/data-warehouse-what-and-why.md

```diff
@@ -0,0 +1,263 @@
+also help to provide evidence about compliance to best practices and guidelines.
+
+For example, we discourage departments from publishing content in PDF format.
+There are many reasons for it, but having some metrics to prove it would
+decrease the distance between the GOV.UK guidelines and department practices
+when publishing content.
+
+In the following example, we can see how there is a relationship between the
+number of PDFs published by `Organisation A` and the `satisfaction score`.
+
+![enforcing good practices][figure-3-comparisons]
+
+### Example 3: Predictions
+
+User interactions with content on GOV.UK can sometimes follow seasonal patterns.
+If on a particular week or month of the year a set of pages receive a spike on
+the number of visits, it is possible those pages will receive a similar number
+of visits the following year during the same period.
+
+Let's explore the following example:
+
+As per the spike of visits in June last year, it is very likely that
+`Organisation A` expects a spike in page views for a content item, so they might
+consider reviewing in advance the content, so it is accurate and up to date.
+
+![predictions][figure-4-predictions]
+
+# Why a Data Warehouse
+
+## State of the art
+
+Before building a new application, we explored if the existing set of
+applications and data sources were good enough to fulfill our mission goals.
+
+This is what we found out:
+
+1. We don't have a single place to retrieve all metrics for content items. We
+   currently store metrics for organizations (and dashboards) via the
+   [performance platform](performance-platform), but the way it is structured
+   and the scope of the product, make it not a good fit for our mission goal.
+
+1. We store the number of page views in [Search][rummager], so when users search
+   in GOV.UK the results are ordered by popularity; the way this information is
+   collected [has been reported as Tech debt][ga-tech-debug-trello] and building
+   this applpcation will enable [Search][rummager] to [use the API][api] and pay
+   the tech debt.
+
+1. Data Scientists find it difficult to get and link content and performance
+   metrics. Having a single place to look for this information will make their
+   daily work easier.
+
+1. Performance Analysts dashboards will also benefit from having metrics from
+   multiple sources because, although they have direct access to metrics from
+   Google Analytics, they don't have direct access to Feedex or other internal
+   applications.
+
+
+We concluded that we had to find custom way to store metrics and content
+attributes through time, because the existing applications were more focused on
+metrics and content characteristics for a single day.
+
+## Tracking metrics through time in a Data Warehouse
+
+Now that we have a better understanding of the needs behind tracking metrics at
+scale and through time, and the state of the art for the existing application,
```

Comment:
scale and through time and ~~the state of the art for the existing~~ need for a new application,